### PR TITLE
Pcs view transfer evidence notes

### PR DESCRIPTION
**Pull request by NateHogan**

### CODE REVIEW

File: src/EA.Weee.Web/Areas/Scheme/Views/ManageEvidenceNotes/ViewAndTransferEvidence.cshtml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+                            <td class="govuk-table__cell">@note.SubmittedBy</td>
+                            @if (note.TypeOfWaste.HasValue)
+                            {
+                                <td class="govuk-table__cell">@(note.TypeOfWaste.Value.ToDisplayString()) </td>
+                            }
+                            else
+                            {
+                                <td class="govuk-table__cell"> </td>
+                            }
+                            <td class="govuk-table__cell">@note.SubmittedBy</td>
```
Comment (meglenapnovoselska) [inline src/EA.Weee.Web/Areas/Scheme/Views/ManageEvidenceNotes/ViewAndTransferEvidence.cshtml:56]: is that correct? is not meant to be a Status?

File: src/EA.Weee.Web/Areas/Scheme/Views/ManageEvidenceNotes/ViewAndTransferEvidence.cshtml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-full">
+            @Html.Partial("_ManageEvidenceNotesHeaderAndTabs", Model)
+
+            <section class="govuk-tabs__panel" id="pcsDetails">
+                <h2 class="govuk-heading-m">
+                    View all reviewed evidence or transfer to another PCS
+                </h2>
+
+                <table id="edit-draft-returned-notes-table" class="govuk-table">
```
Comment (meglenapnovoselska) [inline src/EA.Weee.Web/Areas/Scheme/Views/ManageEvidenceNotes/ViewAndTransferEvidence.cshtml:18]: would you update that?

File: src/EA.Weee.Web/Areas/Scheme/Views/ManageEvidenceNotes/ViewAndTransferEvidence.cshtml [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+                    </thead>
+                    <tbody class="govuk-table__body">
+                        @if (Model.EvidenceNotesDataList.Any())
+                        {
+                            foreach (EditDraftReturnedNote note in Model.EvidenceNotesDataList)
+                            {
+                        <tr class="govuk-table__row">
+                            <td class="govuk-table__cell">@note.ReferenceDisplay</td>
+                            <td class="govuk-table__cell">@(note.SubmittedDate.HasValue ? note.SubmittedDate.Value.ToShortDateString() : "-")</td>
+                            <td class="govuk-table__cell">@note.SubmittedBy</td>
```
Comment (meglenapnovoselska) [inline src/EA.Weee.Web/Areas/Scheme/Views/ManageEvidenceNotes/ViewAndTransferEvidence.cshtml:47]: could be a string empty too? Sorry. I'm not sure

File: src/EA.Weee.Web/Areas/Scheme/Views/ManageEvidenceNotes/ViewAndTransferEvidence.cshtml (hunk 2 above)
Comment (JohnPadgettCivica) [reply src/EA.Weee.Web/Areas/Scheme/Views/ManageEvidenceNotes/ViewAndTransferEvidence.cshtml:18]: Yeah give it another id...

File: src/EA.Weee.Web/Areas/Scheme/Mappings/ToViewModels/ViewAndTransferEvidenceViewModelMap.cs [hunk 4]
```diff
@@ -0,0 +1,39 @@
+﻿namespace EA.Weee.Web.Areas.Scheme.Mappings.ToViewModels
+{
+    using EA.Prsd.Core.Mapper;
+    using EA.Weee.Web.Areas.Aatf.Mappings.ToViewModel;
+    using EA.Weee.Web.Areas.Aatf.ViewModels;
+    using EA.Weee.Web.Areas.Scheme.ViewModels.ManageEvidenceNotes;
+    using System.Linq;
+
+    public class ViewAndTransferEvidenceViewModelMap : IMap<ViewAndTransferEvidenceViewModelMapTransfer, ViewAndTransferEvidenceViewModel>
+    {
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Web/Areas/Scheme/Mappings/ToViewModels/ViewAndTransferEvidenceViewModelMap.cs:10]: ViewAndTransferEvidenceViewModelMap and ReviewSubmittedEvidenceNotesViewModelMap are pretty much exactly the same. We should pull the common code (the stuff that creates the evidencenote list) into its own mapper / or service

File: src/EA.Weee.Web/Areas/Scheme/Mappings/ToViewModels/ViewAndTransferEvidenceViewModelMapTransfer.cs [hunk 5]
```diff
@@ -0,0 +1,27 @@
+﻿namespace EA.Weee.Web.Areas.Scheme.Mappings.ToViewModels
+{
+    using EA.Prsd.Core;
+    using EA.Weee.Core.AatfEvidence;
+    using System;
+    using System.Collections.Generic;
+
+    public class ViewAndTransferEvidenceViewModelMapTransfer
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Web/Areas/Scheme/Mappings/ToViewModels/ViewAndTransferEvidenceViewModelMapTransfer.cs:8]: This looks like the same class as ReviewSubmittedEvidenceNotesViewModelMapTransfer so think we should create an abstract bass class that ReviewSubmittedEvidenceNotesViewModelMapTransfer and ViewAndTransferEvidenceViewModelMapTransfer should inherit from. The base class should have the properties and the child class passes the values to the abstract base class constructor.

File: src/EA.Weee.Web/Areas/Scheme/Views/ManageEvidenceNotes/ReviewSubmittedEvidence.cshtml [hunk 6]
```diff
@@ -40,7 +40,7 @@
                             {
                                 <tr class="govuk-table__row">
                                     <td class="govuk-table__cell">@note.ReferenceDisplay</td>
-                                    <td class="govuk-table__cell">@(note.SubmittedDate.HasValue ? note.SubmittedDate.Value.ToShortDateString() : string.Empty)</td>
+                                    <td class="govuk-table__cell">@(note.SubmittedDate.HasValue ? note.SubmittedDate.Value.ToShortDateString() : "-")</td>
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Web/Areas/Scheme/Views/ManageEvidenceNotes/ReviewSubmittedEvidence.cshtml:43]: Would be tempted to move this display logic to the view model. Then we can test it.

File: src/EA.Weee.Web/Areas/Scheme/Views/ManageEvidenceNotes/ViewAndTransferEvidence.cshtml (hunk 1 above)
Comment (JohnPadgettCivica) [reply src/EA.Weee.Web/Areas/Scheme/Views/ManageEvidenceNotes/ViewAndTransferEvidence.cshtml:56]: > is that correct? is not meant to be a Status?

Yeah looks like it should be

**Review by JohnPadgettCivica — CHANGES_REQUESTED** [review]

File: src/EA.Weee.Web/Areas/Scheme/Mappings/ToViewModels/ViewAndTransferEvidenceViewModelMap.cs (hunk 4 above)
Comment (JohnPadgettCivica) [reply src/EA.Weee.Web/Areas/Scheme/Mappings/ToViewModels/ViewAndTransferEvidenceViewModelMap.cs:10]: @NateHogan I'm going to make a change for this in the branch of code I'm working on. As can see the same code in the AATF.

File: src/EA.Weee.Web/Areas/Scheme/Views/ManageEvidenceNotes/ViewAndTransferEvidence.cshtml (hunk 1 above)
Comment (NateHogan) [reply src/EA.Weee.Web/Areas/Scheme/Views/ManageEvidenceNotes/ViewAndTransferEvidence.cshtml]: Updated

File: src/EA.Weee.Web/Areas/Scheme/Views/ManageEvidenceNotes/ViewAndTransferEvidence.cshtml (hunk 2 above)
Comment (NateHogan) [reply src/EA.Weee.Web/Areas/Scheme/Views/ManageEvidenceNotes/ViewAndTransferEvidence.cshtml]: Given new ID to both tab tables

File: src/EA.Weee.Web/Areas/Scheme/Views/ManageEvidenceNotes/ViewAndTransferEvidence.cshtml (hunk 3 above)
Comment (NateHogan) [reply src/EA.Weee.Web/Areas/Scheme/Views/ManageEvidenceNotes/ViewAndTransferEvidence.cshtml:47]: Originally thought empty was correct but looking at the wireframes, it seems we use - in place of empty values

File: src/EA.Weee.Web/Areas/Scheme/Mappings/ToViewModels/ViewAndTransferEvidenceViewModelMapTransfer.cs (hunk 5 above)
Comment (NateHogan) [reply src/EA.Weee.Web/Areas/Scheme/Mappings/ToViewModels/ViewAndTransferEvidenceViewModelMapTransfer.cs]: Done, I believe

**Review by JohnPadgettCivica — APPROVED** [review]

**Pull request merged**